mozilla-release/other-licenses/nsis/Contrib/CliqzHelper/CliqzHelper.vcxproj

```diff
@@ -66,7 +67,7 @@
     <ClCompile>
       <WarningLevel>Level3</WarningLevel>
       <PrecompiledHeader>NotUsing</PrecompiledHeader>
-      <Optimization>MaxSpeed</Optimization>
+      <Optimization>MinSpace</Optimization>

```

Comment:
Not huge, about 2 kb. I checked parameters for others plugins, they are built with same configurations. Not a big deal here.